include/oneapi/dpl/pstl/tuple_impl.h

```diff
@@ -501,6 +501,15 @@ struct tuple<T1, T...>
         return *this;
     }
 
+    template <typename U1, typename... U>
+    tuple&
+    operator=(const tuple<U1, U...>& other) const
+    {
+        holder.value = other.holder.value;
+        next = other.next;
+        return *this;
+    }
```

Comment:
It looks like it is not `const` in practice, and I don't understand how a const `operator=` makes sense in the first place.
What am I missing?

Reply:
aha yes.  Thanks for the explanation.